packages/ui/src/components/organisms/GroupByScrollList/GroupByScrollList.tsx

```diff
@@ -75,11 +75,15 @@ const GroupByScrollList = ({
       prevItem = data[index - 1];
     }
 
-    if (index === 0 || separatorCondition(prevItem, item)) {
-      const isFirstItem = prevItem == null;
-      const topIndicator = fetchTopIndicator?.(item);
-      const bottomIndicator = !isFirstItem
-        ? fetchBottomIndicator?.(prevItem)
+    const isFirstItem = index === 0;
+    const isLastItem = index === data.length - 1;
+
+    if (isFirstItem || isLastItem || separatorCondition(prevItem, item)) {
+      const topIndicator = !isLastItem ? fetchTopIndicator?.(item) : null;
+      const bottomIndicator =
+        !isFirstItem && !isLastItem ? fetchBottomIndicator?.(prevItem) : null;
+      const lastBottomIndicator = isLastItem
+        ? fetchBottomIndicator?.(item)
         : null;
```

Comment:
When the last item of the list need both bottom and top indicator, it's not working